java-client/src/generated/java/org/opensearch/client/opensearch/_types/ExpandWildcard.java

```diff
@@ -30,32 +30,33 @@
  * GitHub history for details.
  */
 
+//----------------------------------------------------
+// THIS CODE IS GENERATED. MANUAL EDITS WILL BE LOST.
+//----------------------------------------------------
+
 package org.opensearch.client.opensearch._types;
 
+import javax.annotation.Generated;
 import org.opensearch.client.json.JsonEnum;
 import org.opensearch.client.json.JsonpDeserializable;
 
+// typedef: _types.ExpandWildcard
+
 @JsonpDeserializable
+@Generated("org.opensearch.client.codegen.CodeGenerator")
 public enum ExpandWildcard implements JsonEnum {
     /**
-     * Match any data stream or index, including hidden ones.
+     * Match any index, including hidden ones.
```

Comment:
Could we bring back the previous comment? The note about data streams is important I think

Reply:
I'm unable to find any reference to expand wildcard in the context of data streams in any OS docs or in the source code. So I'm not sure if those comments are actually relevant to OS or if they're related to the expanded data stream APIs added in ES.